coss/users/migrations/0002_auto_20170828_0922.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-08-28 09:22
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from wagtail.wagtailcore.models import Site
+
+
+def forwards_default_site(apps, schema_editor):
```

Comment:
Since we are starting from scratch, with a clean db, I am going to drop this data migration.